flowr/src/main/java/com/fueled/flowr/Flowr.java

```diff
@@ -402,6 +411,26 @@ public void close(int n) {
         }
     }
 
+    /**
+     * Closes the current activity if the fragments back stack is empty,
```

Comment:
documentation added says that it pops top `n` fragments, whereas we are popping the stack upto the transaction identified `id`

Reply:
Done! Thanks 👍